src/sim/seq.rs

```diff
@@ -27,8 +27,10 @@ use super::*;
 use super::select::*;
 use super::iterlimit::*;
 use super::earlystopper::*;
-use std::time::Instant;
+use std::boxed::Box;
```

Comment:
I'm not sure why you need this but it might be needed somewhere. Just make sure to remove unused `use` statements if this isn't needed :smile:

Reply:
True laziness on my side. will be gone soon :+1: